packages/salesforcedx-vscode-apex/package.json

```diff
@@ -25,11 +25,11 @@
     "Programming Languages"
   ],
   "dependencies": {
-    "@salesforce/apex-node": "0.2.9",
+    "@salesforce/apex-node": "0.10.0",
     "@salesforce/apex-tmlanguage": "1.4.0",
-    "@salesforce/core": "2.28.0",
+    "@salesforce/core": "^2.35.0",
     "@salesforce/salesforcedx-utils-vscode": "54.2.0",
-    "@salesforce/source-deploy-retrieve": "5.0.0",
+    "@salesforce/source-deploy-retrieve": "5.12.0",
```

Comment:
this is the latest SDR.  Will need to bump 1 more time after readFileSync is updated.